app/src/main/java/com/example/keniphhomemaintenance/CompanionObjects.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.keniphhomemaintenance
+
+import com.example.keniphhomemaintenance.dwellings.Dwelling
+
+class CompanionObjects {
+
+    companion object {
+        const val PLACEHOLDER_ID = 0
```

Comment:
IDs should always at least attempt to be unique - yoink this and just hard-code IDs where you're creating the default data